test/e2e/framework/pod/utils.go

```diff
@@ -338,3 +342,24 @@ func IsPodOnCgroupv2Node(f *framework.Framework, pod *v1.Pod) bool {
 	}
 	return len(out) != 0
 }
+
+func roundUp(in int64) int64 {
```

Comment:
nit: I was going to suggest a more specific name here, but maybe just inline it? If you flip the condition it's simpler, something like

```go
if milliCPULimit%10 != 0 {
	roundedCPULimit = (milliCPULimit/10 + 1) * 10
	// TODO: construct & append rounded limits here
}
```

Reply:
updated